tests/testthat/test-utils.R

```diff
@@ -79,6 +79,11 @@ test_that("get_score_names() works as expected", {
   )
 })
 
+
+# ==============================================================================
+# print
+# ==============================================================================
+
```

Comment:
Instead of having to do this, just define a different [context()] (possibly in a `test-print.R` file)?

Reply:
I looked for `context()`, but it seems like it's deprecated (https://testthat.r-lib.org/reference/context.html). 
<img width="706" alt="image" src="https://github.com/epiforecasts/scoringutils/assets/37978797/9d048148-7159-416d-9f80-9736d527e059">

Basically my idea was just to visually structure the file a bit. 
I didn't create a separate `test-print.R` file so that the names of the test files match the names of the code files. But open to just creating a separate file for the code as well 🤷‍♂️

Reply:
Ah sorry - I'm familiar with the practice of one file per topic. Happy to leave it up to you what you prefer. I guess another question is whether there should be a `R/print.R` file containing print function(s) and/or whether people would expect to find a print function in `R/utils.R` (I don't think I would). That would then naturally lead to a `test-print.R` file following your matching scheme.

Reply:
👍 I added it to this issue #562 to batch renaming and reshuffling files